website/docs/reference/api/legacy/unleash/admin/archive.md

```diff
@@ -4,31 +4,6 @@ title: /api/admin/archive
 
 > In order to access the admin API endpoints you need to identify yourself. Unless you're using the `none` authentication method, you'll need to [create an ADMIN token](/how-to/how-to-create-api-tokens) and add an Authorization header using the token.
 
-### Fetch archived flags {#fetch-archived-toggles}
-
-`GET http://unleash.host.com/api/admin/archive/features`
```

Comment:
Do we need to replace this with another API or there's no other API?

Reply:
An alternative would be: GET `/api/admin/search/features?archived=IS%3Atrue`. That said, my thinking is that since these are legacy docs, this particular detail may not be relevant here.

We've removed a deprecated endpoint, so this particular documentation change should reflect that IMO.

Looping in @melindafekete in case she has thoughts. Happy to follow up in a separate PR if needed.